samples/videoserver/VideoServer2.py

```diff
@@ -30,7 +30,7 @@
 MATRIX_HEIGHT = 32
 FUTURE_DELAY = 5
 URL = "https://youtu.be/WPdCT0l6vGs"
-ESP32_WIFI_ADDRESS = '192.168.8.86'
+ESP32_WIFI_ADDRESS = '192.168.8.127'
```

Comment:
Not really, but as every user of the example would have to change this IP address anyway, it's a bit of a case of six of one, half a dozen of the other.

The actual solution here is making the target IP a script argument, as I think you yourself argued some time ago. Maybe one day this will annoy me to the point I will actually take care of that.